chromadb/test/conftest.py

```diff
@@ -578,29 +600,29 @@ def system_fixtures_ssl() -> List[Callable[[], Generator[System, None, None]]]:
 def system_wrong_auth(
     request: pytest.FixtureRequest,
 ) -> Generator[ServerAPI, None, None]:
-    yield next(request.param())
+    yield from request.param()
```

Comment:
why does this change fix it?

Reply:
cc @tazarov - seems like all the auth fixtures were doing the wrong thing

Reply:
before `next()` was only called once per `request.param()`

but if you call next() once it runs up to the `yield`, you have to call it again for code after the teardown to run

so an equivalent solution would be
```python
def system():
  for r in request.param():
    yield r
```

minimal example to play with: https://www.online-python.com/RVsB9GynFw